src/vs/workbench/contrib/customEditor/browser/customEditorInputFactory.ts

```diff
@@ -124,4 +124,8 @@ export class CustomEditorInputFactory extends WebviewEditorInputFactory {
 			return editor;
 		});
 	}
+
+	public static canResolveBackup(editorInput: IEditorInput, backupResource: URI): boolean {
+		return false;
```

Comment:
Should this use the same implementation that you added for notebooks?

Reply:
Yes, I'll make the change for custom editor.